images/ceph/Dockerfile

```diff
@@ -20,7 +20,7 @@ ARG S5CMD_VERSION
 ARG S5CMD_ARCH
 
 # install 'ip' tool for Multus
-RUN dnf install -y --repo baseos --setopt=install_weak_deps=False iproute && dnf clean all
+# RUN dnf install -y --setopt=install_weak_deps=False iproute && dnf clean all
```

Comment:
can we break this in multiple lines, so then we can understand where this is failing

Reply:
I'm pretty sure the message `failed to solve` is from `dnf install`, so I don't think breaking this up into the 2 commands helps narrow things down much more. I was hoping to see more output, but I didn't find it in the CI runs. 

Given that this succeeds on macos, I suspect this is a CentOS repo/artifact issue affecting amd64 and not arm64. At any rate, `ip` is present in the Ceph image today, but this line is in the dockerfile because we anticipate its eventual removal in the future.